src/main/java/net/minecraftforge/fluids/FluidRegistry.java

```diff
@@ -301,168 +111,16 @@ public static boolean addBucketForFluid(Fluid fluid)
         return ImmutableSet.copyOf(bucketFluids);
     }
 
-
-    public static Fluid lookupFluidForBlock(Block block)
-    {
-        if (fluidBlocks == null)
-        {
-            BiMap<Block, Fluid> tmp = HashBiMap.create();
-            for (Fluid fluid : fluids.values())
-            {
-                if (fluid.canBePlacedInWorld() && fluid.getBlock() != null)
-                {
-                    tmp.put(fluid.getBlock(), fluid);
-                }
-            }
-            fluidBlocks = tmp;
-        }
-        return fluidBlocks.get(block);
-    }
-
-    public static class FluidRegisterEvent extends Event
-    {
-        private final String fluidName;
-        private final int fluidID;
-
-        public FluidRegisterEvent(String fluidName, int fluidID)
-        {
-            this.fluidName = fluidName;
-            this.fluidID = fluidID;
-        }
-
-        public String getFluidName()
-        {
-            return fluidName;
-        }
-
-        public int getFluidID()
-        {
-            return fluidID;
-        }
-    }
-
-    public static int getMaxID()
-    {
-        return maxID;
-    }
-
-    public static String getDefaultFluidName(Fluid key)
-    {
-        String name = masterFluidReference.inverse().get(key);
-        if (Strings.isNullOrEmpty(name)) {
-            FMLLog.log.error("The fluid registry is corrupted. A fluid {} {} is not properly registered. The mod that registered this is broken", key.getClass().getName(), key.getName());
-            throw new IllegalStateException("The fluid registry is corrupted");
-        }
-        return name;
-    }
-
-    @Nullable
-    public static String getModId(@Nullable FluidStack fluidStack)
```

Comment:
This method is still called in `UniversalBucket.java`, causing build failures. Please replace it.